src/main/php/xp/scriptlet/Runner.class.php

```diff
@@ -236,8 +238,22 @@ public function run($url= '/') {
 
       // Service
       $instance->service($request, $response);
-    } catch (\scriptlet\ScriptletException $e) {
-      $cat->error($e);
+    } catch (ScriptletException $e) {
+
+      if (isset($application->logLevels()[$e->getStatus()])) {
+        $logLevel= $application->logLevels()[$e->getStatus()];
+        if (!method_exists($cat, $logLevel)) {
```

Comment:
It would be better if this check was done during configuration time, otherwise this error doesn't surface until an error is handled.

```php
method_exists(\util\log\LogCategory::class, "warn")  // true
method_exists(\util\log\LogCategory::class, "foo")   // false
```